CMake/external_libcurl.cmake

```diff
@@ -1,4 +1,7 @@
 if(CHECK_FOR_UPDATES)
+
+    string(REPLACE "${SECURITY_COMPILER_FLAGS}" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}") # remove flags
```

Comment:
Can you please create a helper function (on another cmake) and call it here..

Something like

`push_remove_security_flags()`

`pop_remove_security_flags()`

And on that cmake you can document the function

Reply:
done, added those 2 + 2 functions for PIC/PIE you mentioned in the other comments